conans/client/rest/conan_requester.py

```diff
@@ -99,18 +99,12 @@ def add_auth(self, url, kwargs):
 class ConanRequester:
 
     def __init__(self, config, cache_folder=None):
-        # TODO: Make all this lazy, to avoid fully configuring Requester, for every api call
-        #  even if it doesn't use it
-        # FIXME: Trick for testing when requests is mocked
-        if hasattr(requests, "Session"):
-            self._http_requester = requests.Session()
-            adapter = HTTPAdapter(max_retries=self._get_retries(config))
-            self._http_requester.mount("http://", adapter)
-            self._http_requester.mount("https://", adapter)
-        else:
-            self._http_requester = requests
-
         self._url_creds = _SourceURLCredentials(cache_folder)
+        self._max_retries = config.get("core.net.http:max_retries", default=2, check_type=int)
```

Comment:
This read of the ``max_retries`` here sounds weird, I guess it is because of preparation for invalidation?

Reply:
Yes, but thinking about it, if the http_requester is invalidated, it would read again from get_max_retries, so actually no need to move this here!